apps/front-end/front-office/src/app/app.routes.ts

```diff
@@ -26,4 +27,8 @@ export const appRoutes: Route[] = [
     path: 'help-page',
     component: HelpPageComponent,
   },
+  {
+    path: 'drag-and-drop',
+    component: DragAndDropComponent,
+  },
```

Comment:
Is there a reason for making a route to this component?

Reply:
To be able to see what I am doing in real time